src/rockstor/storageadmin/models/disk.py

```diff
@@ -40,6 +40,13 @@ class Disk(models.Model):
     vendor = models.CharField(max_length=1024, null=True)
     smart_available = models.BooleanField(default=False)
     smart_enabled = models.BooleanField(default=False)
+    """custom smart options for drive, ie for USB bridges / enclosures"""
+    """eg "-d usbjmicron,p" or "-s on -d 3ware,0"."""
+    smart_options = models.CharField(max_length=64, null=True)
```

Comment:
Is 64 character limit enough for this option? Since it becomes a VARCHAR in postgres, I think it's ok to be a bit generous.